8.8.0-alpha1 Release Notes (#4945)
camunda/camunda-docs: docs/reference/release-notes/880.md

    "software development lifecycle",
    "CI/CD",
    "AI",
  ]
---

These release notes identify the new features included in 8.8, including [alpha feature releases](/components/early-access/alpha/alpha-features.md).

## 8.8 minor

| Scheduled release date | Scheduled end of maintenance | Changelog(s) | Release blog | Update guide |
| ---------------------- | ---------------------------- | ------------ | ------------ | ------------ |
| October 2025           | April 2027                   | -            | -            | -            |

## 8.8.0-alpha1

| Release date    | Changelog(s)                                                                                                                                                                               | Blog                                                                                         |
| :-------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------- |

> | 14 January 2025 | <ul><li>[ Camunda 8 core ](https://github.com/camunda/camunda/releases/tag/8.8.0-alpha1)</li><li>[ Connectors ](https://github.com/camunda/connectors/releases/tag/8.8.0-alpha1)</li></ul> | <!-- [Release blog](https://camunda.com/blog/2025/01/camunda-alpha-release-january-2025) --> |

Reviewer: 🔧 I think this date is not correct :)

Camunda 8 Self-Managed now fully supports the use of Amazon OpenSearch with Optimize.

<!-- https://github.com/camunda/product-hub/issues/2473 -->

### Process instance migration <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span><span class="badge badge--medium" title="This feature affects Zeebe">Zeebe</span>

Enhanced process instance migration allows you to solve problems with process definitions and use the latest process improvements.

You can now migrate the following:

- Compensation boundary event subscriptions
- Escalation boundary events
- Escalation event subprocesses

To learn more about migration, see [process instance migration](/components/concepts/process-instance-migration.md).

<!-- https://github.com/camunda/product-hub/issues/1314 -->

### Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>

A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.

> For more information, see the [Camunda Exporter documentation](/self-managed/zeebe-deployment/exporters/camunda-exporter.md).

Reviewer 2: ```suggestion
To learn more about this feature, see [Camunda Exporter](/self-managed/zeebe-deployment/exporters/camunda-exporter.md).
```
Suggestion for consistency of links on release notes page

> ### Backup and restore changes <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>

Reviewer 2: ```suggestion
### Backup and restore improvements <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
```

> Camunda backups have been overhauled and made easier. The web application backups are merged together under one endpoint.

Reviewer 2: ```suggestion
Camunda backups have been improved and made easier to use. The web application backups are merged together under one endpoint.
```
Wording suggestion
